.github/workflows/tests.yml

```
@@ -0,0 +1,24 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+      - dev
+
+jobs:
+  django-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7.7
+          architecture: x64
+      - name: Install dependencies
+        run: |
+          python -m pip install -e .
+          python -m pip install -r ./requirements.txt
+      - name: Run tests
+        run: |
+          python ./tests.py
```

I'm not sure, but I think the command for running the tests is:
```suggestion
        run:  python manage.py test
```
I don't know if `python ./tests.py` works too.
@johan-smc Can confirm this and provide us with the right command.

---

I saw projects running tests both ways, however I'm sure it depends on the tests. As there are no tests yet, I just took the liberty of choosing that option.